vhdbuilder/packer/packer_source.sh

```diff
@@ -144,9 +144,33 @@ copyPackerFiles() {
 
   if grep -q "kata" <<< "$FEATURE_FLAGS"; then
     # KataCC SPEC file assumes kata config points to the files exactly under this path
-    KATA_CONFIG_DIR=/var/cache/kata-containers/osbuilder-images/kernel-uvm/
+    if [[ $OS_VERSION == "2.0" ]]; then
+      KATA_CONFIG_DIR=/var/cache/kata-containers/osbuilder-images/kernel-uvm
+    elif [[ $OS_VERSION == "3.0" ]]; then
+      KATA_CONFIG_DIR=/usr/share/kata-containers
+    else
+      exit 1
```

Comment:
can we echo an error message here describing the nature of the issue before exiting?